specification/communication/data-plane/Sms/readme.md

```diff
@@ -26,7 +26,21 @@ These are the global settings for the communicationservices.
 
 ```yaml
 openapi-type: data-plane
-tag: package-2020-07-20-preview1
+tag: package-2024-01-14-preview
+```
+
+### Supression
+``` yaml
+suppressions:
+ - code: LroExtension
+   from: communicationservicessms.json
+   reason: Lro attribute was not part of any previous version. Trying to add it signals breaking changes and requires adding operation-location, etc.
+ - code: IntegerTypeMustHaveFormat
+   from: communicationservicessms.json
+   reason: Integer type for HttpStatusCode was not part of any previous version. Trying to add it signals breaking changes.
```

Comment:
We could probably still add it and override the breaking change alert with a label that it improves the swagger to be closer to the deployed service. I don't have a strong opinion on this, maybe @JeffreyRichter ?

Reply:
Adding this to the swagger is fixing a bug because the service already returned it?
If so, then yes, we'd just approve it.

Reply:
@DominikMe so should I remove the LroExtension and IntegerTypeMustHaveFormat suppresions?

Reply:
I believe we still need the LroExtension suppression because of the non-LRO 202 operations. But we can add the integer format, remove the IntegerTypeMustHaveFormat suppression and then let Jeffrey add the BreakingChanges-Approved label.

Reply:
ok added int32 type to SmsResponseItem httpStatusCode.
@JeffreyRichter please review and add Approved-BreakingChange label